docs/sql-migration-guide.md

```diff
@@ -335,6 +335,10 @@ license: |
   
   - Since Spark 3.0, we upgraded the built-in Hive from 1.2 to 2.3. You need to migrate your custom SerDes to Hive 2.3 or build your own Spark with `hive-1.2` profile. See HIVE-15167 for more details.
 
+  - Since Spark 3.0, we upgraded the built-in Hive from 1.2 to 2.3. The result decimal represent in string can be different between Hive 1.2 and Hive 2.3 when use `TRANSFORM` syntax in sql. In Hive 1.2, decimal will eliminate all non-significant zeros in precision when represents in string. But in Hive 2.3, it will always show 18 digits in precision and make up with zero if its precision is less than 18.
```

Comment:
"The string representation of a decimal ..."
use -> using
sql -> SQL
"In Hive 1.2, the string representation omits trailing zeroes. But in Hive 2.3, it is always padded to 18 digits with trailing zeroes if necessary"

Reply:
Let's call it out transform is used for script transformation, which depends on hive's behavior. 

I was not clear how a hive change can affect spark. But they realized that transform is script transformation.